### ISSUE TITLE
Globals are sorted by "Delete" rather than "Name"

### ISSUE THREAD

**Issue by philipstarkey** [labscript-suite-oldfinal1/runmanager#21]
**Original report ([archived issue](https://labscript-suite.github.io/labscript-suite-bitbucket-archive/#!/labscript_suite/runmanager/issues/21)) by Shaun Johnstone (Bitbucket: [shjohnst](https://bitbucket.org/%7B652df738-313d-472c-b7b5-a3f8f98a3322%7D/), GitHub: [shjohnst](https://github.com/shjohnst)).**

----------------------------------------
In qt runmanager, by default, the lists of globals appear to be sorted by the "Delete" column, which makes no sense. In practice this reverts to sorting by "Name" since this is the next column across and the first which contains different values.

It would make more sense to have the arrow indicating which column is being sorted to be above this column rather than the first.

**Comment by philipstarkey** [comment]
**Original comment by Philip Starkey (Bitbucket: [pstarkey](https://bitbucket.org/%7B48af65db-e5fc-459c-a7eb-52eb1f9a5690%7D/), GitHub: [philipstarkey](https://github.com/philipstarkey)).**

----------------------------------------

* set assignee_account_id to "557058:cbf1bc43-1dc2-477b-9e25-1a8f40fd7ee3"
* set assignee to "cbillington (Bitbucket: [cbillington](https://bitbucket.org/cbillington), GitHub: [chrisjbillington](https://github.com/chrisjbillington))"

**Comment by philipstarkey** [comment]
**Original comment by Chris Billington (Bitbucket: [cbillington](https://bitbucket.org/%7Be363c5a9-5075-4656-afb5-88bd6a6dceeb%7D/), GitHub: [chrisjbillington](https://github.com/chrisjbillington)).**

----------------------------------------
Actually the reason they are sorted by name is because I had them inserted in that order:


```
#!python

        globals = runmanager.get_globals({self.group_name: self.globals_file})[self.group_name]
        globals_items = list(globals.items())
        globals_items.sort()  # Ensure globals ordered alphabetically at load time
```

But now I've changed it so that the sorting is done after insertion, with:


```
#!python

        # Sort by name:
        self.ui.treeView_globals.sortByColumn(self.GLOBALS_COL_NAME, QtCore.Qt.AscendingOrder)
```

**Comment by philipstarkey** [comment]
**Original comment by Chris Billington (Bitbucket: [cbillington](https://bitbucket.org/%7Be363c5a9-5075-4656-afb5-88bd6a6dceeb%7D/), GitHub: [chrisjbillington](https://github.com/chrisjbillington)).**

----------------------------------------

* changed state from "new" to "resolved"

Resolves Issue #21: Globals are sorted by "Delete" rather than "Name"

→ \<\<cset 9a97eee969da0d138f300dc462adff4383c81f29\>\>
